stdlib/LinearAlgebra/src/LinearAlgebra.jl

```diff
@@ -582,7 +582,11 @@ function __init__()
     Base.at_disable_library_threading(() -> BLAS.set_num_threads(1))
 
     if !haskey(ENV, "OPENBLAS_NUM_THREADS")
```

Comment:
Wait, this should look at `GOTO_NUM_THREADS` AND  `OMP_NUM_THREADS` as well: https://github.com/xianyi/OpenBLAS/blob/c43ec53bdd00d9423fc609d7b7ecb35e7bf41b85/README.md#setting-the-number-of-threads-using-environment-variables

Reply:
GOTO_NUM_THREADS comes from the original GOTOBLAS library. Nobody should be using it! 

For OMP, I thought that only applies if you build with OpenMP for threading - which we don't. I think we should only look at OPENBLAS_NUM_THREADS.

Reply:
As a matter of fact OpenBLAS respects those variables. If you set them you can see with BLAS.get_num_threads that OpenBLAS changes the setting accordingly

Reply:
Yup it does. I was just saying that we may only want to use one of them and only document that. However, I don't have a strong opinion if we want to use the other ones - but what should we do if more than one are present?

Reply:
If any of those variables is set, we shouldn't override it. See #46118 🙂